dev/openchpl_data-model.sql

```diff
@@ -3201,6 +3202,9 @@ CREATE TABLE openchpl.change_request_status (
     CONSTRAINT certification_body_fk FOREIGN KEY (certification_body_id)
 	    REFERENCES openchpl.certification_body (certification_body_id)
         MATCH SIMPLE ON UPDATE NO ACTION ON DELETE RESTRICT
+    CONSTRAINT user_permission_fk FOREIGN KEY (user_permission_id)
+	    REFERENCES openchpl.user_permission (user_permission_id)
+        MATCH SIMPLE ON UPDATE NO ACTION ON DELETE RESTRICT;
```

Comment:
Is this semi-colon needed? There wasn't one before on the constraint just before it

Reply:
Also, the previous constraint probably needs a comma at the end of line 3204